kerl

```diff
@@ -408,6 +410,7 @@ get_git_releases() {
     tmp="$(mktemp "$TMP_DIR"/kerl.XXXXXX)"
     git ls-remote --tags --refs "$OTP_GITHUB_URL" >"$tmp"
     ret=$?
+    notice "Getting releases from GitHub..."
```

Comment:
Lemme know if this is **too** verbose. There's another one below (for the tarball releases) I'll need to tweak if you want this one removed. (and potentially also the `README`)